### PR TITLE
pkcs11-helper: git rev -> 1.11, openvpn w/ pkcs11-helper support 

### PR DESCRIPTION
**Pull request by Ericson2314**
###### Motivation for this change

New version, and yubi keys with openvpn.
###### Things done

Tested building with and without the feature.
- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [X] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/networking/openvpn/default.nix [hunk 1]
```diff
@@ -13,13 +17,14 @@ stdenv.mkDerivation rec {
...
-    --enable-iproute2
-    IPROUTE=${iproute}/sbin/ip
-  '';
+                  ++ optionals stdenv.isLinux [ pam systemd iproute ]
+                  ++ optional pkcs11Support pkcs11helper;
+
+  configureFlags = optionals stdenv.isLinux [
+    "--enable-systemd"
+    "--enable-iproute2"
+    "IPROUTE=${iproute}/sbin/ip" ''PKCS10_HELPER_LIBS="-lpkcs11-helper-1"'']
```
Comment (rasendubi) [inline pkgs/tools/networking/openvpn/default.nix]: `"PKCS10_HELPER_LIBS=-lpkcs11-helper-1"` would be a bit easier to read.

**Review by rasendubi — APPROVED** [review]

File: pkgs/tools/networking/openvpn/default.nix (hunk 1 above)
Comment (Ericson2314) [inline pkgs/tools/networking/openvpn/default.nix]: Oh oops. That's not supposed to be there at all—pkg-config should work fine.

File: pkgs/development/libraries/pkcs11helper/default.nix [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
+    rev = "${name}";
+    sha256 = "1bfsmy9w2qf7avvs3rsc1ycqczzzw0j2wsqkd2fj4dc1fqzigq2q";
   };
 
   buildInputs = [ pkgconfig openssl autoreconfHook ];
 
   meta = with stdenv.lib; {
     homepage = https://www.opensc-project.org/opensc/wiki/pkcs11-helper;
-    license = with licenses; [ "BSD" gpl2 ];
+    license = with licenses; [ bsd gpl2 ];
```
Comment (Ericson2314) [inline pkgs/development/libraries/pkcs11helper/default.nix]: Should be `bsd3`

**Comment by Ericson2314** [comment]
Sorry for these careless errors, I'll fix and push in ~30 minutes

**Pull request merged**